shtdlib.sh

```diff
@@ -1524,6 +1524,30 @@ function load_config()
     done < "${config_file}";
 }
 
+# Make sure symlink exists and points to the correct target, will remove
+# symlinks pointing to other locations or do nothing if it's correct.
+function ln_sf {
+    target_path="${1}"
+    link_path="${2}"
```

Comment:
Should we return an error if the params aren't provided?

Reply:
Good point.. we can `assert` that both are non-empty

Reply:
I will update to include a quick args count check.